kymatio/frontend/torch_frontend.py

```diff
@@ -0,0 +1,31 @@
+from abc import ABCMeta, abstractmethod
+import torch
+import torch.nn as nn
+from packaging import version
+
+assert version.parse(torch.__version__)>=version.parse("1.3"), 'Current PyTorch version is ' + str(torch.__version__) +\
+                                                    ' . Please upgrade PyTorch to 1.3 at least.'
+
+class ScatteringTorch(nn.Module, metaclass=ABCMeta):
+   def __init__(self):
+       super(ScatteringTorch, self).__init__()
+
+   @abstractmethod
+   def build(self):
+       """ Defines elementary routines. This function should always call and create the filters via
+       self.create_and_register_filters() defined below. For instance, via:
+       self.filters = self.create_and_register_filters() """
+
+   @abstractmethod
+   def create_and_register_filters(self):
```

Comment:
Rename to `create_filters`.

Reply:
I believe in the last mega PR #415 someone said that we should break this function into create filters and register filters. I also believe someone said something about having NumPy and Tensorflow inherit from the same class as they need to create filters, but do not register them.

Reply:
Right. If I remember correctly, I wanted to break it up, @edouardoyallon said that this is just a temporary thing right now, since we hopefully won't have to register filters in the future. If that's the case, I'd rather call this `create_filters` right away instead of this unwieldy name.

Reply:
Sounds good to me.

Reply:
:+1: